src/classes/GE_PaymentServices.cls

```diff
@@ -181,6 +168,14 @@ public with sharing class GE_PaymentServices {
         return createTransaction(requestBody);
     }
 
+//    public UTIL_Http.Response addToCaptureGroup(ElevateTokenizedGift tokenizedGift, String groupId) {
```

Comment:
@dfuller114 - Do we need to keep these commented out methods in this class?

Reply:
good call. just updated.